test/image/mocks/polar_fills.json

```diff
@@ -13,13 +13,37 @@
     "theta": [ 0, 90, 180, 270, 350, 360 ],
     "fill": "toself",
     "hoveron": "fills"
+  }, {
+    "type": "scatterpolar",
+    "mode": "lines",
+    "r": [5, 1, 2, 4, 5],
```

Comment:
This test doesn't show that lines go to the center while markers are not displayed there (though I've seen in playing with it that you indeed got it right 🎉 ) - can you just show markers here, and bump the r=2 value down so no marker *should* show at the center?

Reply:
See https://github.com/plotly/plotly.js/pull/2200/commits/473643cf8dfb12f1543bd8dcb01ca00c568f6398
for what I came up with. I'm not 100% sure this is what you meant though.

Reply:
Yep, you got it, thanks. I was just looking to see a corner in the line at the origin, with no corresponding marker. Lo and behold, there it is!
<img width="175" alt="screen shot 2018-01-12 at 5 53 38 pm" src="https://user-images.githubusercontent.com/2678795/34900931-9326709c-f7c1-11e7-9818-512bfc1bcc05.png">